gradle/libs.versions.toml

```diff
@@ -1,31 +1,31 @@
 [versions]
-ktx = "1.10.1"
-viewmodel-lifecycle = "2.6.1"
+ktx = "1.12.0"
```

Comment:
Bumping ktx version to 1.12.0 will fail because it was compiled against 34, our application was compiled against 33.